resources/lang/nl/fieldtypes.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+return [
+    'array' => [
+        'config' => [
+            'keys' => 'Stel de array keys (variabelen) en optionele labels in.',
+            'mode' => 'Dynamische modus geeft de gebruike rcontroler over de data terwijl keyed modus dat niet doet.',
+        ],
+    ],
+    'assets' => [
+        'config' => [
+            'allow_uploads' => 'Nieuwe file-uploads toestaan',
+            'container' => 'Kies welke asset-container je voor dit veld wilt gebruiken.',
+            'folder' => 'De map waarin de gebruiker begint met bladeren.',
+            'max_files' => 'Het maximaal aantal selecteerbare assets.',
+            'mode' => 'Kies je voorkeur voor layoutstijl.',
+            'restrict' => 'Voorkom dat gebruikers naar andere mapen kunnen bladeren.',
+        ],
+    ],
+    'bard' => [
+        'config' => [
+            'allow_source' => 'Schakel de optie in dat gebruikers de HTML-broncode kunnen bekijken tijdens het schrijven.',
+            'buttons' => 'Kies welke knoppen er in de toolbar zitten.',
+            'container' => 'Kies welke asset-container er voor dit veld gebruikt moet worden.',
+            'fullscreen' => 'Schakel de optie in dat mensen voleldige schermweergave kunnen gebruiken.',
+            'link_noopener' => 'Gebruik `rel="noopener"` op alle links.',
+            'link_noreferrer' => 'Gebruik `rel="norefferer"` op alle links.',
```

Comment:
TYPO: norefferer => noreferrer

Reply:
Cool. Is it possible to edit a PR? I’d like to fix those asap.

Reply:
Yes Rob, push again to the same branch

Reply:
Good to know, push to my own forked branch, will try in a minute.